### ISSUE TITLE
Unable to use three finger swipe with voiceover to swipe left or right.

### ISSUE THREAD

**Issue by anikasetia** [archriss/react-native-snap-carousel#597]
### Is this a bug report, a feature request, or a question?

Question

I am trying to swipe on the card using a three finger swipe while using voice over on iOS. I am not able to do so. Is there a way to implement this.
### Have you followed the required steps before opening a bug report?

(Check the step you've followed - put an `x` character between the square brackets (`[]`).)

- [] I have read [the guidelines regarding bug report](https://github.com/archriss/react-native-snap-carousel/blob/master/CONTRIBUTING.md).
- [x] I have reviewed [the documentation](https://github.com/archriss/react-native-snap-carousel/blob/master/README.md) in its entirety, including the dedicated documentations :books:.
- [x] I have searched for [existing issues](https://github.com/archriss/react-native-snap-carousel/issues) and made sure that the problem hasn't already been reported.
- [x] I am using [the latest plugin version](https://github.com/archriss/react-native-snap-carousel/releases).
- [] I am following [the issue template](https://raw.githubusercontent.com/archriss/react-native-snap-carousel/master/ISSUE_TEMPLATE.md) closely in order to produce a useful bug report.

### Have you made sure that it wasn't a [React Native bug](https://github.com/archriss/react-native-snap-carousel/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22react-native+bug%22+)?

Yes

### Is the bug specific to iOS or Android? Or can it be reproduced on both platforms?

(Write your answer here and specify the iOS/Android versions on which you've been able to reproduce the issue.)

### Is the bug reproductible in a production environment (not a debug one)?

(Write your answer here.)

### Environment

Environment:
    "react-native": "^0.59.9",
    "react-native-snap-carousel": "^3.8.0",

Target Platform:
   iOS 12
   iOS 13.1.2
### Expected Behavior

To mimic single finger swipe when voice over isn't on

### Actual Behavior

Nothing happens

### Reproducible Demo

(Paste the link to a [Snack example](https://snack.expo.io/) in which the issue can be reproduced. Please follow [the guidelines](https://stackoverflow.com/help/mcve) for providing a Minimal, Complete, and Verifiable example.)

### Steps to Reproduce

(Write your steps so that anyone can reproduce the issue in the Snack demo you provided.)

1.
2.
3.

**Comment by bd-arc** [comment]
@anikasetia Are you able to get this working with a regular `FlatList`?

**Comment by anikasetia** [comment]
I was able to swipe and scroll flatlist and scrollview using 3 fingers

**Comment by kalistara** [comment]
@bd-arc I have the same issue and ```FlatList``` is working correctly.

Environment:
"react-native": "0.61.2",
"react-native-snap-carousel": "3.8.4"

Also I had to manually set accessible status to renderItems. All renderItems were accessible and not only the current renderItem.

**Comment by dohooo** [comment]
Sorry, please allow me to advertise for my open source library! ~
I think this library [react-native-reanimated-carousel](https://github.com/zwh1666258377/react-native-reanimated-carousel) will solve your problem. It is a high performance and very simple component, complete with React-Native reanimated 2
